update 6.0 release notes dm compatibility (#8128)
pingcap/docs: releases/release-6.0.0-dmr.md

    - Because of internal mechanism changes, the interface related to task management cannot be compatible with the previous experimental version. You need to refer to the new [DM OpenAPI documentation](/dm/dm-open-api.md) for adaptation.
- DM changes the methods to resolve conflicts during the full import phase
    - A `loader.<name>.on-duplicate` parameter is added. The default value is `replace`, which means using the new data to replace the existing data. If you want to keep the previous behavior, you can set the value to `error`. This parameter only controls the behavior during the full import phase.

> - DM must use with the corresponding `dmctl` CLI

Reviewer: ```suggestion
- DM must use with the corresponding version of `dmctl`
```

>     - Because internal mechanism changes, after you upgrade DM to v6.0.0, you must upgrade `dmctl` to the same version.

Reviewer 2: ```suggestion
- To use DM, you should use the corresponding `dmctl` CLI
    - Due to internal mechanism changes, after upgrading the DM to v6.0.0, you should also upgrade `dmctl` to v6.0.0.
```
Does the same version indicate v6.0.0?

> - DM must use with the corresponding version of `dmctl`
>     - Because internal mechanism changes, after you upgrade DM to v6.0.0, you must upgrade `dmctl` to the same version.

Author: ```suggestion
- To use DM, you should use the corresponding version of `dmctl`
    - Due to internal mechanism changes, after upgrading DM to v6.0.0, you should also upgrade `dmctl` to v6.0.0.
```
